stage1/units/units/halt.target

```diff
@@ -1,4 +0,0 @@
-[Unit]
-Description=Halt
-DefaultDependencies=no
-AllowIsolate=yes
```

Comment:
It's generated programmatically in https://github.com/coreos/rkt/pull/3218/files#diff-99ef63dd92c607458ff979e0c391eb56R68